src/renderer/pages/Operations/components/LogModal.tsx

```diff
@@ -35,11 +36,20 @@ const EventMessage: Partial<Record<SigningStatus | 'INITIATED', string>> = {
   ERROR_CANCELLED: 'log.errorCancelledMessage',
 } as const;
 
+const getFilteredWalletsMap = (wallets: Wallet[]): Map<Wallet['id'], Wallet> =>
+  new Map(
+    wallets
+      .filter((w) => walletUtils.isValidSignatory(w) || w.type === WalletType.MULTISHARD_PARITY_SIGNER) // 2nd condition for legacy multisig
+      .map((w) => [w.id, w]),
+  );
+
 const LogModal = ({ isOpen, onClose, tx, account, connection, contacts, accounts }: Props) => {
   const { t, dateLocale } = useI18n();
   const { getLiveTxEvents } = useMultisigEvent({});
-  const walletsMap = new Map(useUnit(walletModel.$wallets).map((w) => [w.id, w]));
-  const accountsMap = new Map(useUnit(walletModel.$accounts).map((a) => [a.accountId, a]));
+  const filteredWalletsMap = getFilteredWalletsMap(useUnit(walletModel.$wallets));
+  const filteredAccountMap = new Map(
```

Comment:
To reduce logic clutter in UI it's nice to make similar function as `getFilteredWalletsMap`